code/game/objects/items/theft_tools.dm

```diff
@@ -30,6 +31,9 @@
 		return ..()
 
 /obj/item/nuke_core/process()
+	var/turf/T = loc
+	if(T in blacklisted_locs)	// no radiation when in proper containment
```

Comment:
Remove the T, replace this with `if(is_type_in_typecache(loc, blacklisted_locs_typecache)`